tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -1969,6 +1969,14 @@ Status ConvertConv2DHelper(OpConverterParams* params, int group,
   } else {
     padding = {{0, 0}, {0, 0}};
   }
+
+// TensorRT 5.1 added support for asymmetric padding. Due to a bug in 5.1.2, we
+// can only use asymmetric padding in convolutions with 5.1.3+.
+#if (NV_TENSORRT_MAJOR > 5) ||                           \
```

Comment:
Please use `IS_TRT_VERSION_GE(5, 1, 3, 0)` here and below.

Reply:
Oh cool, did not see that before.